pkg/scripts/install.sh

```diff
@@ -74,7 +74,7 @@ verify_configuration_file() {
 
   # Make sure the config values are defined
   set +e
-  INVALID_CONTENT=$(grep '<' $CONFIG_FILE & grep '>' $CONFIG_FILE)
+  INVALID_CONTENT=$(grep -o '^[^#]*' $CONFIG_FILE | grep '<\|>')
```

Comment:
shell noob question, what does this line do? I made some similar changes in my [PR](https://github.com/atlassian-labs/data-center-terraform/pull/52) Happy to remove it if this resolve the issue